src/main/scala/dpla/ingestion3/mappers/providers/MdlMapping.scala

```diff
@@ -39,6 +42,15 @@ class MdlMapping extends JsonMapping with JsonExtractor with IdMinter[JValue] wi
   override def dplaUri(data: Document[JValue]): ExactlyOne[URI] =
     new URI(mintDplaId(data))
 
+  override def edmRights(data: Document[json4s.JValue]): ZeroToOne[URI] =
+    extractString(unwrap(data)  \\ "record" \ "rights") match {
+      case Some(t) => Some(new URI(t))
+      case None => None
+    }
+
+  override def intermediateProvider(data: Document[_root_.org.json4s.JsonAST.JValue]): ZeroToOne[EdmAgent] =
```

Comment:
Question about the data input value: I notice that here the value `data` is defined as `Document[_root_.org.json4s.JsonAST.JValue]`, while in the method above it is `Document[json4s.JValue]` and below it is `Document[JValue]`.  What is the difference?